protocol/standard-l2-genesis.md

```diff
@@ -95,24 +95,36 @@ block matches the expected value.
 
 #### SuperchainConfig "Upgrader" Role
 
-This new role is to allow for operational flexibility going into the future. If its not desired to add this role,
-we could make the `guardian` able to faciliate the upgrade transactions. In practice, we may end up setting both
-the `guardian` and the `upgrader` to the same account for the Superchain.
-
-Open to removing this from the spec and just using the `guardian` if preferred, but the possible operational
-flexibility seems useful.
-
 The `upgrader` role can call the `OptimismPortal.upgrade(bytes memory data, uint32 gasLimit)` function
 and it emits a deposit tx from the `DEPOSITOR_ACCOUNT` that calls the `L2ProxyAdmin`. Sourcing the auth
 from the `SuperchainConfig` allows for simple management of this very important role, given that it impacts
-stage 1 status. This is meant to simplify operations, ie remove the concept of the aliased L1 `ProxyAdmin` owner
+stage 1 status. This is meant to simplify operations by removing the aliased L1 `ProxyAdmin` owner
 being set as the L2 `ProxyAdmin`.
 
+Since the the L1 and L2 `ProxyAdmin` contracts are intended to have the same owner,
+an additional improvement (which may be excluded to limit scope creep), would be to remove the
+`Ownable` dependency on the L1 `ProxyAdmin` contract, and instead have it read the
+`SuperchainConfig.upgrader()` role to authorize upgrades.
```

Comment:
I do like this, but one issue is that it prevents a chain from deploying with their own upgrader to start, and only later transferring that role

Reply:
TODO: timeboxed product investigation into the implications of this change.

Reply:
I synced up with @maurelian on this over slack. Chiming in with the product perspective: 

This seems strictly positive. We don't really want the SC to be pausing chains that they're not also responsible for upgrading, and participation in the pause is a benefit that chains get when they join the Superchain ecosystem.

Reply:
Great, thanks, I added a not to reflect this: 9b3732bdd8daf5834b7e46ea271d401e1b80ecac